### PR TITLE
feat: update display font sizes and line heights

### PR DESCRIPTION
**Pull request by rgraber**
Update all display font sizes and line heights to match design docs
Screen Shot 2021-03-31 at 3 58 43 PM

### CODE REVIEW

File: scss/core/_variables.scss [hunk 1]
```diff
@@ -1,4 +1,5 @@
 // Variables
+// Variables
```
Comment (ciduarte) [inline scss/core/_variables.scss:2]: Was this a mistake?

**Review by ciduarte — APPROVED** [review]
I'm not sure what these values are supposed to be, but LGTM + one question

File: scss/core/_variables.scss (hunk 1 above)
Comment (rgraber) [reply scss/core/_variables.scss:2]: yes, and I couldn't figure out what about it changed

Comment (abutterworth) [reply scss/core/_variables.scss:2]: // Våriables ?

**Review by abutterworth — APPROVED** [review]

**Pull request merged**